src/components/PeoplePage/PeoplePage.jsx

```diff
@@ -0,0 +1,35 @@
+import React, { useEffect, useState } from 'react';
+import { getPeople } from '../api/people';
+import { PeopleTable } from '../PeopleTable/PeopleTable';
+
+export const PeoplePage = () => {
+  const [people, setPeople] = useState([]);
+
+  useEffect(() => {
+    getPeople()
```

Comment:
You could fetch `people` from the server right directly in the `PeopleTable` component, instead of passing this array in props